.github/workflows/_docker_build.yaml

```diff
@@ -19,6 +19,9 @@ on:
       image:
         description: The image name and tag.
```

Comment:
This will need adjusting given that we added the `tag` input, right?

Reply:
Seems that the `output.image` is not used after the change. Do you think we should remove this output item?

Reply:
If it's unused then yes, sure

Reply:
removed the `outputs.image` in the docker build workflow.